src/login/states/trello/view.jade

```diff
@@ -1,7 +1,12 @@
 div.landing-page(layout="row" layout-align="center center")
   div.hgroup
     h1 Not So Shitty
-    h3 Automate your daily boring tasks
+    h3
+      a(
+        href="http://attackwithnumbers.com/the-laws-of-shitty-dashboard"
+        target="_blank"
+      ) The law of shitty dashboards
```

Comment:
what do you think about it? I was trying to remove as much as possible from the menu
![screenshot from 2015-12-21 23-24-40](https://cloud.githubusercontent.com/assets/551837/11942876/0c791df2-a83a-11e5-899b-765b1ba4a658.png)